src/core/server/metrics/logging/get_ops_metrics_log.ts

```diff
@@ -30,10 +30,29 @@ export function getEcsOpsMetricsLog(metrics: OpsMetrics) {
   // HH:mm:ss message format for backward compatibility
   const uptimeValMsg = uptimeVal ? `uptime: ${numeral(uptimeVal).format('00:00:00')} ` : '';
 
-  // Event loop delay is in ms
+  // Event loop delay metrics are in ms
   const eventLoopDelayVal = process?.event_loop_delay;
   const eventLoopDelayValMsg = eventLoopDelayVal
-    ? `delay: ${numeral(process?.event_loop_delay).format('0.000')}`
+    ? `mean delay: ${numeral(process?.event_loop_delay).format('0.000')}`
```

Comment:
Now that we have the histogram percentiles in the log message, we need to let folks know what measure the single delay is.

Reply:
> I wonder if it would be more readable to combine the values in the existing delay, similar to how it is structured in the API.

We could but that would be a breaking change for folks ingesting the log message. ATM, if someone was parsing the message string they might expect `delay: number` and end up with `delay: [Object]`. 

Admittedly, this PR also changes the message string but keeps the delay type the same. Appending the additional data seems like a less risky option here.

Admittedly, ingesting log message data isn't the best approach and folks should be using the `meta` instead. However, we removed `meta` from all the logs expect for `debug` mode so the extra info wouldn't be available for other levels.

Reply:
> We could but that would be a breaking change for folks ingesting the log message.

Yeah, although I suppose we are adding `mean delay` so a grok pattern against just `delay` could match two results now (both mean and histogram). However, in the past we have not considered the message format to be stable, and have made breaking changes to it in minors anyway. So I am not super concerned about this either way.